operator/src/lgen/mod.rs

```diff
@@ -0,0 +1,9 @@
+/// Lgen controller arm module for reconciling load generator resources.
+#[cfg(feature = "controller")]
+pub mod controller;
+/// Job module for creating load generator jobs.
+#[cfg(feature = "controller")]
+pub mod job;
+/// Spec module for creating load generator specs.
+#[cfg(feature = "controller")]
+pub mod spec;
```

Comment:
Want to also add these couple of lines at the end? Very minor, just for consistency with the other mods:
```
#[cfg(feature = "controller")]
pub use controller::run;
```